lib/main.go

```diff
@@ -214,7 +214,20 @@ func (obj *Main) Run() error {
 		return fmt.Errorf("hostname cannot be empty")
 	}
 
-	var prefix = fmt.Sprintf("/var/lib/%s/", obj.Program) // default prefix
```

Comment:
Wouldn't it be better to leave this first line as-is, and then If state dir, use that, else if xdg is set, use that, and so on?

Reply:
Oh I guess I misunderstood. I thought you wanted to use XDG_CACHE_HOME instead of /var/lib/mgmt. The reason I did it that way was because if XDG_CACHE_HOME isn't set it's supposed to be $HOME/.cache. I know at least on Fedora (what I'm using) it's not set as an environment variable out of the box so you would need to consciously add it. I can take out the default of $HOME/.cache and leave the /var/lib/%s line at the top.

Reply:
Maybe we should think about what makes the most sense... Hmmm

Reply:
How about if root, then that's the default (/var/lib/ ...) otherwise the default is XDG. And either can be overridden by the systemd state var.

What do you think? (Don't necessarily just agree with me, decide yourself =D)

Reply:
Yeah that sounds like it makes sense. I'm not sure how useful XDG dirs are for root anyway since you normally don't run an X session as root.